docs/Projects/TAD/tools_comparison.md

```diff
@@ -0,0 +1,79 @@
+# Comparison of tools for transparency of algorithmic decision making
+
+[We have researched](tools.md) a few tools which we want to investigate
+further, this document is the next step in that investigation.
+We created a [checklist](index.md) to compare these tools against.
+The Fulfilled column will give a numerical value based
+on the priority which can then be compared with the other tools per section.
+The scoring is as follows: {M:4, S:3, C:2, W:-1}.
```

Comment:
Arguably, I am nitpicking now, but the description of the way how you score is not complete. Maybe something like:
```suggestion
The scoring is as follows. Each requirement is assigned a value between 0 and a maximum depending on the priority; the maxima are defined as follows {M:4, S:3, C:2, W:-1}.
```

Reply:
I didn't completely take over your suggestion, but made it for sure more clear. Do you agree with the new one?